EIPS/eip-TBD.md

```diff
@@ -1,85 +1,57 @@
-
-  Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
-
-  The title should be 44 characters or less. It should not repeat the EIP number in title, irrespective of the category.
-
-  TODO: Remove this comment before submitting
--->
-
 ## Abstract
 
-We describe a mechanism for which, upon transfer of a Non-Fungible Token (NFT), the dominant strategy of the new owner is to truthfully reveal if the owning entity has changed and, if so, report their valuation of the token and pay a creator-specified royalty based on said valuation.
-
-<!--
-  The Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
-
-  TODO: Remove this comment before submitting
--->
+We describe a mechanism for which, upon transfer of an [ERC-721](./eip-721.md) Non-Fungible Token (NFT), the dominant strategy of the new owner is to truthfully reveal if the owning entity has changed and, if so, (a) report their valuation of the token and (b) pay a creator-specified royalty based on said valuation.
 
 ## Motivation
 
 While [ERC-2981](./eip-2981.md) introduced a standard for contracts to signal a royalty amount to be paid to the creator upon sale of an NFT, it doesn't describe a means of enforcement.
 It goes so far as to state that "payment must be voluntary, as transfer mechanisms… \[do\] not always imply a sale occurred".
 
-The problem with on-chain enforcement stems from the fact that the information available to any enforcement contracts is incomplete.
-So, upon token transfer, contracts are unaware of the nature of the transfer (e.g. sale, private, gift, etc.) or any corresponding flow of funds (e.g. sales price), which are commonly needed to compute the amount of funds that has to be paid to the creator.
-Contracts therefore have to rely on external parties to supply this information which can either be gamed if the involved parties are dishonest or introduce centralisation.
-This problem is aggravated in OTC deals, where the flow of funds can be handled entirely via separate channels that are unknown or intransparent to any third party.
+The difficulty with on-chain enforcement stems from the incomplete information available to enforcement contracts.
+Such contracts are unaware of both the nature of token transfers (e.g. sale, private, gift, etc.) and any corresponding flow of funds (e.g. sales price), which are commonly needed to compute the royalty to be paid to the creator.
+Until now, contracts have therefore relied on external parties to supply this information, which can either introduce centralisation, or be manipulated if the involved parties are dishonest.
```

Comment:
Not sure about the "until now". This sentence was intended as a consequence of the former.

Reply:
Removed. I'm leaving the rest as is so it's both a consequence of the former and also a reference to the historical status quo.